### DISCUSSION TITLE
Update 4.3 ----Shutter dosen.t work

### DISCUSSION THREAD

**Discussion by Maxym1979** [Blackymas/NSPanel_HA_Blueprint#1942] [Q&A]
Hello

I successfully updated my NS panel from 4.1.1 to 4.3.
Unfortunately now I have the following problem: my shutter control no longer works.
As soon as I press the Hardbutton and trigger the scene, the Scene will immediately be disabled in HA
I have the same problem when I run this as a soft button
With version 4.1.1 everything worked.
Do you have any idea why an active scene is set to deactivated in HA?

Thank you in advance for the help.

HA Version 
Core 24.3.1
OS 12.1
ESP Home 2024.2.2

**Comment by edwardtfn** [comment]
- #1945
